source/examples/login.js

```diff
@@ -60,9 +60,19 @@ async function loginAndFetch() {
     await session.handleIncomingRedirect(window.location.href);
   }
 
-  // 4. Make authenticated requests by passing `session.fetch` to solid-client functions. For example:
-  const myDataset = await getSolidDataset(session.info.webId, { fetch: session.fetch });
+  // 4. Make authenticated requests by passing `session.fetch` to solid-client functions.
+  // The user must have logged in as someone with the appropriate access to the specified URL.
+  
+  // For example, the user must be someone with Read access to the specified URL.
+  const myDataset = await getSolidDataset("https://docs-example.inrupt.net/profile/card", { fetch: session.fetch });
+
   // ...
+  
+  // For example, the user must be someone with Write access to the specified URL.
+  const savedSolidDataset = await saveSolidDatasetAt(
+    "https://docs-example.inrupt.net/profile/card",
+    myUpdatedDataset,
```

Comment:
Since "updated" can also be interpreted as "saved to the Pod", I wonder if this might avoid that?

```suggestion
    myChangedDataset,
```